Code/Java/CircularQueue

```diff
@@ -0,0 +1,402 @@
+
+/**
+ * Author - Khushee Kapoor
```

Comment:
Hey, In open source, we donot add our names while contributing codes :)

Reply:
Made the changes.